core/templates/components/profile-link-directives/profile-link-image-backend-api.service.ts

```diff
@@ -32,8 +34,10 @@ export class ProfileLinkImageBackendApiService {
     private http: HttpClient
   ) {}
 
-  fetchProfilePictureData(profileImageUrl: string): Promise<ProfileDict> {
-    return this.http.get<ProfileDict>(profileImageUrl).toPromise();
+  fetchProfilePictureData(profileImageUrl: string): Promise<string> {
+    return this.http.get<ProfileDict>(profileImageUrl).pipe(
+      map(response => decodeURIComponent(
```

Comment:
Add a comment to explain why we are doing this.

Reply:
Done!